src/brain.js

```diff
@@ -0,0 +1,33 @@
+const { Client } = require("pg");
+
+let client;
+const brain = new Map();
```

Comment:
Previously, we used Hubot which had a `brain` plugin that stored stuff in redis. Since we needed to reimplement this "brain" behavior, I switched us to postgres on the assumption it would be cheaper to maintain.

Reply:
Now that Redis will be AWS Elasticache, whether Postgres will be cheaper to maintain than Redis is a developer call, as the first 10 nodes of Redis are not charged in the cloud.gov pricing.